### PR TITLE
new icon: supabase (original, original-wordmark, plain, plain-wordmark)

### PR DESCRIPTION
**Pull request by PantheraRed**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes NONE**
#952 

## Link to prove your SVG is correct and up-to-date.
https://supabase.com/
https://supabase.com/brand-assets

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -8922,6 +8922,24 @@
...
+        "tags": [
+            "cloud-computing-platform",
+            "database",
+            "authentication",
+            "storage"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
```
Comment (Panquesito7) [inline devicon.json:8937]: Please add these to the font versions, or create a font version of the icon.
Let us know if you need any help with that. 🙂

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: devicon.json (hunk 1 above)
Comment (PantheraRed) [reply devicon.json:8937]: I was having some trouble while making them so I thought I'd do them after the pull request.

Comment (Snailedlt) [reply devicon.json:8937]: I agree with @Panquesito7 :)
If you're really fast you'll get them in the upcoming release

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — DISMISSED** [review]
Thanks! 🚀

image

image

File: devicon.json [hunk 2]
```diff
@@ -7323,7 +7323,7 @@
             ]
         },
         "color": "#3775a9",
-        "aliases": []  
+        "aliases": []
```
Comment (lunatic-fox) [inline devicon.json:7326]:
```suggestion
        "aliases": []  
```

File: devicon.json [hunk 3]
```diff
@@ -7311,7 +7311,7 @@
             "programming"
         ],
         "versions": {
-            "svg": [ 
+            "svg": [
```
Comment (lunatic-fox) [inline devicon.json:7314]:
```suggestion
            "svg": [ 
```

**Review by lunatic-fox — COMMENTED** [review]
Remove some modifications not related to this entry in `devicon.json`.

**Review by lunatic-fox — APPROVED** [review]
Perfect icons and great work! ✔
Thank you for this contribution! 🚀

**Comment by Panquesito7** [comment]
> Remove some modifications not related to this entry in `devicon.json`.

Should I fix those by committing directly or?

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**